tests/externs.el

```diff
@@ -59,3 +59,47 @@
 ;;;TODO
 ;; ensure that any symbols bound with the import syntax are added to the extern list
 ;; ensure that any symbols bound with the export syntax exist in the file scope
+
+;; Test `/*jslint*/` declarations
+
+(ert-deftest js2-finds-jslint-declaration-externs ()
+  (with-temp-buffer
+    (insert "/*jslint browser, node:false, es6: true */")
```

Comment:
I thought I read somewhere that it had a similar meaning to something like `/*global foo: false*/`, where the assumed externs would all be read-only, but I can't find that anywhere, so maybe I dreamed it. ;)

I removed those bits from the tests and did some other improvements.